static/css/settings.css

```diff
@@ -268,6 +274,7 @@
 
 .developer-checkbox-item,
 .experiment-item,
+.localization-item,
```

Comment:
Nit: This background colour is too dark to form a good contrast with the dropdown. Should be `#5288af`, as per:

```
.network-item,
.wireless-network-item,
.user-item,
.adapter-item,
.addon-item,
.authorization-item,
.domain-item {
  background-color: #5288af;
}
```